keras_nlp/src/utils/preset_utils.py

```diff
@@ -546,7 +548,12 @@ def load_config(preset, config_file=CONFIG_FILE):
     return config
 
 
-def validate_metadata(preset):
+def check_format(preset):
+    if check_file_exists(preset, SAFETENSOR_FILE) or check_file_exists(
+        preset, SAFETENSOR_CONFIG_FILE
+    ):
+        return "transformers"
```

Comment:
@mattdangerw this does not account for the fact if a hugging face repository has safetensors and a h5 file, as you can see in [google-bert/bert-base-uncased](https://huggingface.co/google-bert/bert-base-uncased/tree/main).

The current tests fail becasue there is yet to be a port written for the bert. Do you think having a check for `METADATA_FILE` first, and then checking for safetensors make more sense here?

Reply:
To me it seems like everything is works as intended here, we just need to tweak the tests.

We have four tests that were added for showing a friendly error if someone is trying to load a transformers checkpoint we do not support....

https://github.com/search?q=repo%3Akeras-team%2Fkeras-nlp+%22hf%3A%2F%2Fgoogle-bert%2Fbert-base-uncased%22&type=code

I think we just need to change to error expectation, instead of a `FileNotFound` error we will get a `ValueError`. Eventually, we might want to delete those tests and consolidate all of this "transformers testing" into the new directory you are adding, but that can be a follow up I think.

Reply:
So no changes here right?

Reply:
I think we still need to update the error here, to a `ValueError`. https://github.com/keras-team/keras-nlp/blob/e0efbc8ce23429d1ba49099902fac7bd3ab6ca27/keras_nlp/src/models/backbone_test.py#L53-L55